Sources/Sentry/SentryFileManager.m

```diff
@@ -66,7 +66,7 @@ - (NSString *)uniqueAcendingJsonName {
                                       [NSUUID UUID].UUIDString];
 }
 
-- (NSArray<SentryFileContents *> *)getAllEvents {
+- (NSArray<SentryFileContents *> *)getAllEventsAndMaybeEnvelopes {
```

Comment:
```suggestion
- (NSArray<SentryFileContents *> *)getAllEventsAndEnvelopes {
```

Reply:
l: having `Maybe` seems odd, it's fine to return no envelopes if there are none.
It might have no events as well, so it'd be getMaybeAllEventsAndMaybeEnvelopes?

Reply:
I just wanted to make it clear for the consumer of this method. If I leave the name `getAllEvents` most likely somebody is going to run into trouble. See this comment for more info: https://github.com/getsentry/sentry-cocoa/blob/philipp/rate-limit-for-cached/Sources/Sentry/include/SentryFileManager.h#L29-L34
I'm happy about suggestions on how to improve this.

Reply:
mm I'm running out of ideas here too, it's a `l` so feel free to ignore, but having `Maybe` on its name feels weird to me, but I get your point.